Metro-Records/la-metro-councilmatic
# Update rules and procedure pdf

## Overview

This branch updates the rules and procedures pdf on the About Us page from an external link to a locally hosted pdf. It also merges a couple of the the sections on the same page

- Closes #1097 
- Closes #1098

## Testing Instructions

 * Head to the About Us page
 * Confirm that the "Board Room Rules" and "Rules & Procedures" sections have been merged
 * Click the link that reads "Read about all Metro Rules & Procedures."
 * Confirm that a locally hosted pdf opens, identical to the one described in the issue
